app/models/protected_area.rb

```diff
@@ -147,6 +147,18 @@ def overlap(pa)
     overlap
   end
 
+  def self.global_terrestrial_oecm_coverage
+    land_oecms = ProtectedArea.oecms.marine_areas.pluck(:reported_area)
```

Comment:
@stanleypliu Here you are now fetching marine areas instead of terrestrial. If there's not terrestrial scope you can create one. Also, since you are already in the `ProtectedArea` class, you can use `self` instead of `ProtectedArea`, as `self` will refer to the class itself given these are class methods

Reply:
😓 that's what I get for not checking after I copy and paste